Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestJobsNode.java

```diff
@@ -35,6 +36,13 @@
 import org.openide.util.NbBundle.Messages;
 import org.sleuthkit.autopsy.datamodel.NodeProperty;
 import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestJob.Stage;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestAdminActions;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestDashboard;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestMonitor;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestNodeRefreshEvents;
+import org.sleuthkit.autopsy.experimental.autoingest.Bundle;
+import org.sleuthkit.autopsy.experimental.autoingest.Manifest;
+import org.sleuthkit.autopsy.experimental.autoingest.PrioritizationAction;
```

Comment:
was looking around in the code for the name of the _aiaa file and noticed these added imports appear to be unnecessary as they are all from the same package.

Reply:
I believe codacy calls those out

Reply:
Thanks, fixed in next commit.